### DISCUSSION TITLE
Tell us about your use of Posit tools

### DISCUSSION THREAD

**Discussion by kmasiello** [posit-conf-2024/ds-workflows-r#31] [General]
Hello Workshop participants! 
In anticipation of the Workshop on Monday, we're curious to hear about which (if any) of Posit's open source or professional tools (Workbench, Connect, Package Manager) you may already be using in your organization. We'd love to hear about the kind of work you do and how you're using these tools in your day to day. 

As for me, before coming to Posit five years ago, I was using RStudio daily, typically doing statistical analysis and documenting my results with R Markdown. I was also beginning to work on building internal packages. I definitely hit the package management wall of reproducibility pain and would have benefitted from Package Manager + `renv`.  I used Connect a bit for scheduled jobs, but most notably for sending an email to a colleague and myself every morning with a randomly generated encouraging message :) 

Now I use the Posit tools every day, and I really enjoy helping teams adopt workflows and best practices that reduce friction and let their insights and best work have impact.

**Comment by sanjmeh** [comment]
I use rstudio daily.  Build Shiny apps regularly. Use many packages. I'd request you to list the Posit tools available so that I can identify them.

**Reply by doh-FXX0303** [reply]
I am still using Rstudio Desktop daily and have not been able to use Posit tools at the moment. I have dabbled in Posit Connect to publish a markdown with Shiny data visualization --this was mainly for instructive purposes and not used in anything else. I would love to hear and see more tools that I can use.

**Reply by tnederlof** [reply]
Most of the R work and packages being used in this course can still be used without Posit Workbench and Connect. Connect can help with the emailing and hosting of APIs/Apps but there are alternatives.

**Comment by paulditterline** [comment]
Hi Everyone! At Heaven Hill we use Posit Team every day to help make your bourbon better 🥃. 

We rely heavily on Workbench to build *lots* of things and Posit Connect to host for the business to consume.

**Reply by tnederlof** [reply]
Very cool! What kind of things do you use Connect for?

**Reply by paulditterline** [reply]
We host collaborative notebooks for data engineering and data science to build data pipelines, host statistical reports that the business requests, shiny applications across the company for business consumption, and are getting ready to start hosting models as plumber APIs for our ERP to consume!

**Reply by paulditterline** [reply]
I've used Posit Team at two companies, if you have questions or wonder about the usefulness I'm happy to connect with you!

**Comment by TheDataShrink** [comment]
I can say that Posit (RStudio) changed my life. I use RStudio and VS code every day. Just started with Positron as it feels that is a great unification of the two tools. I am setting myself a task for this year to learn observable and quarto as much as possible. Got introduced to GitHub actions. Thanks for such an amazing work done to get us here. PS: if you think you want to keep in touch have a look at this new website that I am trying to bring to life using R and Quarto. www.thedatashrink.com (not done yet...hope to get it up and running soon)

**Reply by tnederlof** [reply]
Very cool! I will say getting into GitHub Actions and CI/CD in general was a big moment for me to automate parts of my workflows. We won't get into CI/CD in this workshop but happy to chat with anyone if they are interested!

**Comment by sparrow925** [comment]
I am using RStudio Desktop for my day to day work. I am trying to build out team workflows that work with our data lake infrastructure (we are using SkySpark for building data) and am excited to get insights on best practices from this material. (I had to swap to remote at the last minute)
